src/hooks/useGetQuestion.ts

```diff
@@ -13,10 +14,14 @@ export function useCreateGame() {
       .get(`${API_URL}/games/question/${code}`)
       .then(response => {
         console.log(response);
+        const question = response.data;
+        if (question) {
+          dispatch(submitQuestion(question));
```

Comment:
I know we haven't added the right dispatchers / data in QuestionState yet, but just for clarity, we will need to change this to not be "Submit Question" as soon as possible. If that is not going to be part of this PR, we should just console.log the response.data along with a message like "got question!". `submitQuestion` should only be dispatched when a user is actually answering a question. If we include this dispatch,  it will submit an answer as soon as the question is fetched, causing the useSubmitAnswer hook to run (bad)

Reply:
Done